implementations/rust/ockam/ockam_vault/src/secret_impl.rs

```diff
@@ -88,6 +88,7 @@ impl SoftwareVault {
 #[async_trait]
 impl SecretVault for SoftwareVault {
     /// Generate fresh secret. Only Curve25519 and Buffer types are supported
+    #[tracing::instrument(skip_all, err)]
```

Comment:
Is this preventing from sensitive data leakage to tracing surface?

Reply:
I don't think so, but I don't think it implements debug.

Reply:
@thomcc I'm just wondering what is the purpose of this here

Reply:
I can remove it if you care, but it was useful when debugging.

Reply:
I mean, if we don't have clear reasons to keep it - let's remove it